chart/kubeapps/values.yaml

```diff
@@ -508,6 +508,12 @@ dashboard:
   ##      background-color: #991700;
   ##    }
   customStyle: ""
+  ## Relative path to custom components js bundle
+  ## Note: The bundle file MUST be called 'custom_components.js'
```

Comment:
mmh, I think it's not needed to be called `custom_components.js`, the Helm function will just read whatever file you pass it.